### PR TITLE
issue #79. init enhacement

### PR DESCRIPTION
**Pull request by umneeq**
Now it writes error message when directory does not exist

### CODE REVIEW

File: init [hunk 1]
```diff
@@ -173,8 +173,13 @@ function getParams()
 function setWritable($root, $paths)
 {
     foreach ($paths as $writable) {
-        echo "      chmod 0777 $writable\n";
-        @chmod("$root/$writable", 0777);
+        if(is_dir("$writable")) {
```
Comment (cebe) [inline init]: should be `is_dir("$root/$writable")`

Comment (samdark) [inline init]: Won't it fail if not chmodded first?

Comment (cebe) [inline init]: why should it? or what do you mean with "it"?

Comment (samdark) [inline init]: I mean `is_dir`.

Comment (cebe) [inline init]: if current user is not able to write the dir it can not chmod it. the chmod is only to allow writing for other users e.g. webserver user.

**Pull request closed without merging**